Sources/NIO/Socket.swift

```diff
@@ -240,6 +240,46 @@ typealias IOVector = iovec
         }
     }
 
+    func recvmsg(pointer: UnsafeMutableRawBufferPointer,
+                 storage: inout sockaddr_storage,
+                 storageLen: inout socklen_t,
+                 controlBytes: inout Slice<UnsafeMutableRawBufferPointer>,
+                 controlMessagesReceived: inout UnsafeControlMessageCollection?) throws -> IOResult<Int> {
```

Comment:
Is it really better to have two of these rather than an optional parameter to the above method? Do we even ever call the above method now?